### ISSUE TITLE
Preact-cli yarn global install failing

### ISSUE THREAD

**Issue by jrock2004** [developit/preact-cli#142]
I have made the switch from npm to yarn. I am trying to install, on a new machine, preact cli and getting error about a dependency. 

```
sudo yarn global add preact-cli 
yarn global v0.24.6
warning No license field
[1/4] Resolving packages...
[2/4] Fetching packages...
warning @google-cloud/functions-emulator@1.0.0-alpha.21: The engine "node" is incompatible with this module. Expected version ">=6.9.1".
info "@google-cloud/functions-emulator@1.0.0-alpha.21" is an optional dependency and failed compatibility check. Excluding it from installation.
warning fsevents@1.1.2: The platform "linux" is incompatible with this module.
info "fsevents@1.1.2" is an optional dependency and failed compatibility check. Excluding it from installation.
error less-loader@4.0.4: The engine "node" is incompatible with this module. Expected version ">= 4.3 < 5.0.0 || >= 5.10".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/global for documentation about this command.
```

Has anyone got this to install using yarn?

**Comment by lukeed** [comment]
Hi @jrock2004 !

One of the differences between Yarn & NPM is that Yarn will throw & quit on version mistmatch, whereas NPM will emit an error but allow the installation to continue.

So, what's happening here is that you've installed `functions-emulator`, which requires [`node >= 6.9`](https://github.com/GoogleCloudPlatform/cloud-functions-emulator/blob/master/package.json#L8). Yarn is throwing an error and exiting because your active node version (`node -v`) does not satisfy this requirement.

It's basically trying to let you know _early_ that dependencies (`functions-emulator` in this case) won't be able to run/compile in your current Node environment.

TL;DR: This is Yarn telling you to upgrade Node if you want to use the dependency :)

**Comment by jrock2004** [comment]
The version of nodejs was too old. Upgraded it and now I am ok. Sorry for the bad ticket
